src/commands/bundle/index.ts

```diff
@@ -0,0 +1,27 @@
+import { buildCommand, getConfigFilePath } from '../../utils';
+import BuildCommand from '../build';
+
+export default class BundleCommand extends BuildCommand {
+  public static description = 'Bundle source code';
+
+  public static examples = [`$ sl-scripts bundle`];
+
+  protected get commands() {
+    const parsed = this.parse(BundleCommand);
+
+    return [
+      buildCommand(`rollup --config ${getConfigFilePath('rollup.config.js')}`, {
+        rawArgs: parsed.raw,
+        flags: Object.keys(BundleCommand.flags),
+      }),
```

Comment:
So if I understand correctly rollup will transpire TypeScript to JavaScript and then use Tsc just to get the declarations out, right?

Reply:
Yeah, that's right.
The old rollup plugin emitted declarations for us, but the new one doesn't support it yet.
As soon as they add support for emitting declarations, we can remove the tsc command